pop-api/integration-tests/src/fungibles/utils.rs

```diff
@@ -1,16 +1,5 @@
 use super::*;
 
-fn do_bare_call(function: &str, addr: &AccountId32, params: Vec<u8>) -> ExecReturnValue {
```

Comment:
comment for other reviewers. These were moved to here: pop-api/integration-tests/src/utils.rs